packages/sdk/src/fetcher/search/index.ts

```diff
@@ -0,0 +1,23 @@
+import { KPIToken } from "../../entities/kpi-token";
+import { KPITokensObjectProp, transformInKPITokensObject } from "../serializer";
+
+export function searchKPItokens(
+    searchQuery: string,
+    allKPITokens: KPITokensObjectProp
+) {
+    const filteredTokens = Object.values(allKPITokens).filter((token) => {
+        return tokenSpecificationIncludesQuery(token, searchQuery);
+    });
+
+    return transformInKPITokensObject(filteredTokens);
+}
+
+function tokenSpecificationIncludesQuery(token: KPIToken, searchQuery: string) {
+    const query = searchQuery.toLowerCase();
+    return (
+        token.hasOwnProperty("specification") &&
+        (token.specification.title.toLowerCase().includes(query) ||
+            token.specification.description.toLowerCase().includes(query) ||
+            Object.values(token.specification.tags).includes(query))
```

Comment:
I think we should also make the tags lowercase `token.specification.tags.toString().toLowerCase()`.
Also, is `Object.values()` actually needed?

Reply:
Done